cmd/fluxd/main.go

```diff
@@ -128,6 +128,7 @@ func main() {
 		logger = log.With(logger, "ts", log.DefaultTimestampUTC)
 		logger = log.With(logger, "caller", log.DefaultCaller)
 	}
+	logger.Log("started", true)
```

Comment:
Structured logging, it's the way of the future ..

Reply:
*whispers* logrus